### PR TITLE
Ft contact form

### PR DESCRIPTION
**Pull request by felixodette**
For the third milestone of my portfolio website, I:
* Added a contact form to your portfolio and the portfolio of your coding partner.
* Made sure that my users' experience is in a way that helps them fill in the form without frustration.
* Added footer with social icons.
* Completed this part of the project with my coding partners.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -666,3 +713,183 @@
...
+*/
+
+.get-started {
+  position: absolute;
+  width: 320px;
+  height: 813px;
+  left: -2px;
+  top: 4229px;
+  background: #f9f9fa;
+}
```
Comment (keroloussamy) [inline css/style.css]: - [ ] Kindly here change the `width: 320px` to `width: 100%` to fill the whole width, to match the figma design. 
- [ ] Kindly make the content of the section in the center to match the Figma design.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @felixodette, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
Comment (felixodette) [reply css/style.css]: Hi @keroloussamy! Thanks for the quick reply 😄. I have:

- [ ] Attempted to put the width of the section to 100%, that spans over and above the 320px width container that's. In its stead, I've constricted all elements to be within the 320px and centered the form, 16px on either side of it.
Screenshot 2022-03-29 at 17 20 25

It does seem to work on the inspector in mobile view though. Changes made and committed. Thanks!

**Review by OLIPLICHE — APPROVED** [review]
Hi @felixodette , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**